### ISSUE TITLE
PIDLock drivetrain while shooting

### ISSUE THREAD

**Issue by kooryan** [TheGreenMachine/Zodiac#82]

**Comment by andrewfhou** [comment]
Just be really really careful about your current draw here, and how aggressive your control loop is

**Comment by kooryan** [comment]
Agreed, this will be very hard to do without causing any damage to the robot, so most likely will not be of top priority as of now.
